docker-compose.yml

```diff
@@ -14,24 +14,5 @@ nomad-web:
     - NOMADBOARD_SECRET_KEY=1337+dev+secret+key
 
 nomad-db:
-  container_name: nomad-postgres
+  container_name: nomad-db
   image: postgres
-  volumes_from:
-    - nomad-data
-
-nomad-grunt:
```

Comment:
Any reason why you removed this? Without this is't no possible to run grunt because it doesn't install npm dependencies.